tracks/elm/exercises/twelve-days/mentoring.md

```diff
@@ -0,0 +1,41 @@
+---
+
+This is a good exercise to talk about naming, as it is quite difficult and most students struggle with it.
+
+This exercise introduces some List handling functions, such as `List.map`, `List.range`, `List.take` and `List.reverse`, and some string handling functions, such as `++` and `String.join`.
+
+Students should strive to communicate what the code does in as few lines as possible (ideally you should be able to understand what is it trying to do in the entry function). They show also strive to communicate how it does it in as few lines as possible.
+
+### Reasonable solutions
+
+Simply listing out the complete text of all verses is the simplest and easiest solution, and would be a reasonable initial choice in real life. Then, if changes were required to the song, it would make sense to refactor to remove the duplication, and then make the changes. However this solution doesn't really help the student learn elm, so if they have done this it makes sense to encourage them to make a further iteration.
+
+All solutions need a way of turning an `Int` in to a string ordinal, and in to a string describing the gift for a day. `Dict.fromList`, `Array.fromList`, `List` or `case statements` are the normal ways to do this, and there is not much to choose between them.
```

Comment:
```suggestion
All solutions need a way of turning an `Int` into a string ordinal, and into a `string` describing the gift for a day. `Dict.fromList`, `Array.fromList`, `List` or `case statements` are normal ways to do this, and are equally favourable.
```

I didn't understand the last , `[...and, ... between them]`. Is this what you meant?

Reply:
Thanks for the simplification!